docs/protocol.md

```diff
@@ -72,23 +72,23 @@ See [DID Create API](#DID-Creation) section for detail on how to construct a cre
 
 DIDs may include attached values that are used in resolution and other activities. The standard way to pass these values are through _DID Parameters_, as described in the [W3C DID spec](https://w3c.github.io/did-core/#generic-did-url-parameters).
 
-Many DID Methods feature a period of time (which may be indefinite) between the generation of an ID and the ID being anchored/propagated throughout the underlying trust system (i.e. blockchain, ledger). The community has recognized the need for a mechanism to support resolution and use of identifiers during this period. As such, the community will introduce a _Generic DID Parameter_ `initial-values` that any DID method can use to signify initial state variables during this period. 
+Many DID Methods feature a period of time (which may be indefinite) between the generation of an ID and the ID being anchored/propagated throughout the underlying trust system (i.e. blockchain, ledger). The community has recognized the need for a mechanism to support resolution and use of identifiers during this period. As such, the community will introduce a standardized DID parameter `-<method-name>-initial-state` that any DID method can use to signify initial state variables during this period.
```

Comment:
Consider rename "standardized" to "convention using method specific", since anything that's not specified in spec is arguably not standard.